Cargo.toml

```diff
@@ -35,7 +35,7 @@ features=["bevy_core_pipeline", "bevy_render", "bevy_asset", "png", "bevy_winit"
 [target.'cfg(unix)'.dev-dependencies.bevy]
 version = "0.8"
 default-features = false
-features=["bevy_core_pipeline", "bevy_render", "bevy_asset", "png", "bevy_winit", "x11"]
+features=["bevy_core_pipeline", "bevy_render", "bevy_asset", "png", "bevy_winit", "x11", "filesystem_watcher"]
```

Comment:
The "filesystem_watcher" feature should go under the dev-dependencies. I know that wasm, android, and iOS likely don't support this feature but our examples are meant to run primarily on desktop. :) If you feel strongly about this let's add new config targets for windows and macos to make sure they also get the "filesystem_watcher" feature. Thank you!

Reply:
No, this is a fair point. Let's add it to dev-dependencies :)